praw/reddit.py

```diff
@@ -556,8 +585,7 @@ def patch(
             of the request (default: None).
 
         """
-        data = self.request("PATCH", path, data=data)
-        return self._objector.objectify(data)
+        return self._objectify_request("PATCH", path, data=data)
```

Comment:
For each `_objectify_request` call, please pass all arguments via keyword and sort them.